code/clement/html_css_flask/lab01/static/styles/index.css

```diff
@@ -0,0 +1,95 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Cuprum:wght@500&family=Vazirmatn&display=swap');
+
+*{
+    margin: 0;
+    padding: 20;
+    /* width: 100%; */
+    height: auto;
+}
```

Comment:
Using `*` can be slow. Consider creating a container to hold the entire page that would then represent *.

Reply:
I believe `height: auto;` is the default setting and therefore might not be required here. No need to change, just pointing it out.